Maple2.Model/Metadata/Table/MasteryRewardTable.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System.Collections.Generic;
+using Maple2.Model.Enum;
+
+namespace Maple2.Model.Metadata;
+
+public record MasteryRewardTable(IReadOnlyDictionary<MasteryType, MasteryRewardTable.Entry> Entries) : Table {
```

Comment:
this can be a nested dictionary `IReadOnlyDictionary<MasteryType, IReadOnlyDictionary<int, Entry>>`

There's an extension method I wrote that makes reading from these easier: https://github.com/kOchirasu/Maple2/blob/master/Maple2.Tools/Extensions/EnumerableExtensions.cs#L67